### PR TITLE
Update mentions protobuf

### PR DESCRIPTION
**Pull request by typfel**
## What's new in this PR?

Update mentions protobuf which replace start/end with start/length

### CODE REVIEW

File: Protos/Messages.pb.m [hunk 1]
```diff
@@ -1,20 +1,4 @@
...
-//
-// This program is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-// GNU General Public License for more details.
-//
-// You should have received a copy of the GNU General Public License
-// along with this program. If not, see http://www.gnu.org/licenses/.
-//
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```
Comment (vytis) [inline Protos/Messages.pb.m]: Copyright header gone!

File: Protos/Messages.pb.m [hunk 2]
```diff
@@ -1,20 +1,4 @@
-//
```
Comment (typfel) [inline Protos/Messages.pb.m]: Wrong copyright header!

**Comment by typfel** [comment]
@zenkins  release major

**Comment by typfel** [comment]
@zenkins release major

**Pull request merged**